routes/project.ts

```diff
@@ -116,15 +116,20 @@ async function zipApkAndKey(signedApkPath: string, pwaSettings: PwaSettings, sig
     postfix: ".zip"
   });
   const fileStream = fs.createWriteStream(zipFile); 
-  zipStream.pipe(fileStream);
-  await zipStream
-    .file(signedApkPath, { name: apkName })
-    .file(signingKey.keyStorePath, { name: "signing-keystore.keystore" })
-    .file("./Next-steps.md", { name: "Next-steps.md" })
-    .append(signingKey.keyStorePassword, { name: "key-store-password.txt" })
-    .append(signingKey.keyPassword, { name: "key-password.txt" })
-    .append(signingKey.keyAlias, { name: "key-alias.txt" })
-    .finalize();
+  try {
+    zipStream.pipe(fileStream);
+    await zipStream
+      .file(signedApkPath, { name: apkName })
```

Comment:
Maybe it's worth adding the generated `twa-manifest.json` to the zip file? Might help debugging the values that went into the APK.

Reply:
Good idea. I'll check with the team, but I think this should be fine.